source/docs/beta/beta-getting-started/index.rst

```diff
@@ -0,0 +1,11 @@
+Beta Getting Started
+======================
```

Comment:
```suggestion
====================
```

Reply:
I definitely didn't do this on purpose and agree with fixing it but this is also not in the styleguide and conflicts with Sphinx documentation about being "at least as long"